fuseutil/file_system.go

```diff
@@ -93,6 +94,12 @@ type fileSystemServer struct {
 	opsInFlight sync.WaitGroup
 }
 
+type opCtx struct {
+	c   *fuse.Connection
+	ctx context.Context
```

Comment:
Embedding context.Context in structs is an anti-pattern: https://groups.google.com/forum/#!topic/golang-nuts/xRbzq8yzKWI